packages/data/src/onboarding/action-types.js

```diff
@@ -3,6 +3,7 @@ const TYPES = {
 	SET_IS_REQUESTING: 'SET_IS_REQUESTING',
 	SET_PROFILE_ITEMS: 'SET_PROFILE_ITEMS',
 	SET_TASKS_STATUS: 'SET_TASKS_STATUS',
+	SET_PAYMENT_METHODS: 'SET_PAYMENT_METHODS',
```

Comment:
Totally optional since this data store already contains our old style of action semantics, but same comment as here - https://github.com/woocommerce/woocommerce-admin/pull/6918#discussion_r625403389

I think `GET_PAYMENT_METHODS_SUCCESS` or `PAYMENT_METHODS_RECEIVED` would be more in line with Redux's recommendations.

Reply:
Nice, I've changed it to `GET_PAYMENT_METHODS_SUCCESS`. However, I've yet to follow the whole nine yards (`REQUEST` & `ERROR` variants).